src/ui/templates/results/universalresults.hbs

```diff
@@ -1,12 +1,15 @@
-{{#if isSearchComplete}}
-  {{#if showNoResults}}
-    {{> results/noresults}}
-  {{else}}
-    <div class="universal-result-list">
-      <div class="universal-result-sections"
-          data-component="VerticalResults"
-          data-prop="sections">
+<div class="yxt-ResultsContainer
```

Comment:
could we add this css class to `universal-result-list` instead? We have to be really careful about changing the DOM structure of SDK components.

Reply:
I had originally had it this way but I changed it to a wrapper because currently there are no dom objects if a search is not complete. I'm not sure if adding a wrapper or having `universal-result-list` permanently be in the dom was better. @tmeyer2115 should I be worried about leaving `universal-result-list` in the dom in all phases of the search? Also Should I be adding the class to `results/noresults` as well?